data/js/twemoji.min.js

```diff
@@ -0,0 +1,2 @@
+/*! Copyright Twitter Inc. and other contributors. Licensed under MIT */
```

Comment:
Please don't include `twemoji.min.js` in the repository. It is cleaner to install it with Bower.